src/main/java/com/conveyal/gtfs/validator/NewTripTimesValidator.java

```diff
@@ -166,6 +180,19 @@ private void processTrip (List<StopTime> stopTimes) {
         // All bad references should have been recorded at import and null trip check is handled above, we can just
         // ignore nulls.
         Route route = routeById.get(trip.route_id);
+        if (route != null &&
+            (route.continuous_drop_off == 0 ||
+            route.continuous_drop_off == 2 ||
+            route.continuous_drop_off == 3 ||
+            route.continuous_pickup == 0 ||
+            route.continuous_pickup == 2 ||
+            route.continuous_pickup == 3)
+        ) {
+            hasContinuousBehavior = true;
+        }
```

Comment:
This is repeat code with `stopTime`. I would extract a method `hasContinuousBehavior(continuousDropOff, continuousPickup)` to perform the check.

Reply:
Updated.